src/Microsoft.PowerShell.Commands.Management/commands/management/WebServiceProxy.cs

```diff
@@ -296,11 +296,7 @@ private string GenerateRandomName()
                 }
             }
 
-            string sequenceString;
-            lock (s_sequenceNumberLock)
-            {
-                sequenceString = (s_sequenceNumber++).ToString(CultureInfo.InvariantCulture);
```

Comment:
`(s_sequenceNumber++).ToString(CultureInfo.InvariantCulture)` actually calls `ToString` on the original value of `s_sequenceNumber`, and then increment `s_sequenceNumber`. For example, running the following c# code:
```c#
	uint a = 1;
	string c = (a++).ToString();
	Console.WriteLine(c);
	Console.WriteLine(a);
```
you will get
```console
1
2
```
However, after your change, the incremental will happen before `ToString`, and that changes the original behavior.